tests/dummy/app/styles/app.scss

```diff
@@ -1 +1 @@
-@import '@gavant/ember-validations';
```

Comment:
@billdami shouldn't this be `@gavant`?

Reply:
@bakerac4 nah, that was an error in the previous release. we are importing the sass partial using its filename here, which is `gavant-ember-validations.scss`.